components/list/list-item-drag-mixin.js

```diff
@@ -98,3 +99,84 @@ export const ListItemDragMixin = superclass => class extends superclass {
 		` : nothing;
 	}
 };
+
+export class NewPositionEventDetails {
+	/**
+	 * @param { Object } object An simple object with the position event properties
+	 * @param { String } object.targetKey The item key of the list-item that is moving
+	 * @param { String } object.destinationKey The item key of the list-item in the position we are moving to
+	 * @param { String } object.temporaryMovement Information on whether the item is entering or exiting temporary movement
+	 */
+	constructor(object) {
+		this.targetKey = object.targetKey;
```

Comment:
at which point validation of object properties should happen? How it is guaranteed that the constructor will get valid object at a runtime?

Reply:
I can add validation to the constructor; this is the responsibility of the functions dispatching the event within the mixin.

Reply:
Not sure if this object will know if the key is valid or can know.